### PR TITLE
Add SEI support for Grants Stack

### PR DESCRIPTION
**Pull request by codenamejason**
Fixes: #3225 (partially)

## Description

Updates wallet configs and supported chains with the SEI network on:

- [x] Builder
- [x] Explorer
- [x] Manager

## Checklist

This PR:

- [x] Does it add new payout or donation token? In this case, have they been added to the indexer to avoid it to crash?
- [x] Includes only changes relevant to the original ticket. Significant refactoring needs to be separated.
- [x] Doesn't disable eslint rules.
- [x] Doesn't work around the type checker (including but not limited to: type casts, non-null assertions, `@ts-ignore`, unjustified optional values).
- [x] Doesn't contain commented out code.
- [x] If adding/updating a feature, it adds/updates its test script on Notion.

### CODE REVIEW

File: packages/builder/.env.example [hunk 1]
```diff
@@ -85,11 +85,15 @@ REACT_APP_SUBGRAPH_SEPOLIA_API=”https://api.thegraph.com/subgraphs/name/bharga
 REACT_APP_SUBGRAPH_SCROLL_API="https://api.studio.thegraph.com/query/45391/grants-round-scroll/v0.0.2"
 REACT_APP_SUBGRAPH_DEV1_API=http://localhost:8545
 REACT_APP_SUBGRAPH_DEV2_API=http://localhost:8546
+REACT_APP_SUBGRAPH_SEI_DEVNET_API=test
```
Comment (thelostone-mc) [inline packages/builder/.env.example:88]:
```suggestion
REACT_APP_SUBGRAPH_SEI_DEVNET_API=test
```

File: packages/builder/.env.example [hunk 2]
```diff
@@ -85,11 +85,15 @@ REACT_APP_SUBGRAPH_SEPOLIA_API=”https://api.thegraph.com/subgraphs/name/bharga
...
 
 # --------------------------------
 # Optional for local development
 # --------------------------------
 
+#  Overrride the default chains
+REACT_APP_CHAINS_OVERRIDE=all
+
```
Comment (thelostone-mc) [inline packages/builder/.env.example:96]:
```suggestion
```

File: packages/round-manager/.env.example [hunk 3]
```diff
@@ -53,6 +53,7 @@ REACT_APP_SUBGRAPH_ZKSYNC_TESTNET_API="https://api.thegraph.com/subgraphs/name/g
 REACT_APP_SUBGRAPH_ZKSYNC_MAINNET_API="https://api.studio.thegraph.com/query/45391/grants-round-zkera/v0.0.2"
 REACT_APP_SUBGRAPH_SEPOLIA_API=”https://api.thegraph.com/subgraphs/name/bhargavaparoksham/grants-round-sepolia-testnet”
 REACT_APP_SUBGRAPH_SCROLL_API="https://api.studio.thegraph.com/query/45391/grants-round-scroll/v0.0.2"
+REACT_APP_SUBGRAPH_SEI_DEVNET_API=
```
Comment (thelostone-mc) [inline packages/round-manager/.env.example:56]:
```suggestion
```

File: packages/round-manager/.env.test [hunk 4]
```diff
@@ -21,6 +21,7 @@ REACT_APP_PINATA_JWT=token-test
 REACT_APP_PINATA_BASE_URL=http://localhost
 REACT_APP_SUBGRAPH_SEPOLIA_API="https://api.thegraph.com/subgraphs/name/bhargavaparoksham/grants-round-sepolia-testnet"
 REACT_APP_SUBGRAPH_SCROLL_API="https://api.studio.thegraph.com/query/45391/grants-round-scroll/v0.0.2"
+REACT_APP_SUBGRAPH_SEI_DEVNET_API=test
```
Comment (thelostone-mc) [inline packages/round-manager/.env.test:24]:
```suggestion
```

**Review by thelostone-mc — APPROVED** [review]

**Comment by michellema1208** [comment]
@thelostone-mc / @codenamejason - I'm seeing SEI Devnet support on prod as well - I'm assuming we only want it to be available on staging? 

Additionally - @codenamejason would be super helpful if you can share some testing guidelines here! Are we still blocked on Lit support? What should we test in particular?

**Comment by codenamejason** [comment]
> @thelostone-mc / @codenamejason - I'm seeing SEI Devnet support on prod as well - I'm assuming we only want it to be available on staging?

SEI only has a live devnet currently.

> 
> Additionally - @codenamejason would be super helpful if you can share some testing guidelines here! Are we still blocked on Lit support? What should we test in particular?

Test Flows:

- [ ] Create a project
- [ ] Create a program
- [ ] Create a round

**Review by michellema1208 — APPROVED** [review]
tested creating a program + round + project, per Jason's guidelines above! All are working -- amazing work! I assume we won't merge this PR until Lit issue is resolved though? As we can't test the rest of the flow

**Comment by thelostone-mc** [comment]
> tested creating a program + round + project, per Jason's guidelines above! All are working -- amazing work! I assume we won't merge this PR until Lit issue is resolved though? As we can't test the rest of the flow

synced internally, we'll be merging this PR to prevent the code from getting stale.
Once we migrate to new LIT we'll raise a follow up PR to get the last piece working

**Pull request merged**